src/avif.c

```diff
@@ -181,14 +181,14 @@ avifResult avifImageCopy(avifImage * dstImage, const avifImage * srcImage, avifP
     avifImageSetMetadataXMP(dstImage, srcImage->xmp.data, srcImage->xmp.size);
 
     if ((planes & AVIF_PLANES_YUV) && srcImage->yuvPlanes[AVIF_CHAN_Y]) {
+        if ((srcImage->yuvFormat != AVIF_PIXEL_FORMAT_YUV400) &&
+            (!srcImage->yuvRowBytes[AVIF_CHAN_U] || !srcImage->yuvRowBytes[AVIF_CHAN_V])) {
```

Comment:
I am wondering if it would be better to test `(!srcImage->yuvPlanes[AVIF_CHAN_U] || !srcImage->yuvPlanes[AVIF_CHAN_V])` instead. We test `srcImage->yuvPlanes[AVIF_CHAN_Y]` at line 183 and `srcImage->alphaPlane` at line 193.